controllers/ocsinitialization/ocsinitialization_controller.go

```diff
@@ -316,14 +329,7 @@ func (r *OCSInitializationReconciler) SetupWithManager(mgr ctrl.Manager) error {
 					Name:      util.RookCephOperatorConfigName,
 					Namespace: r.OperatorNamespace,
 				},
-			},
-			handler.EnqueueRequestsFromMapFunc(
-				func(context context.Context, obj client.Object) []reconcile.Request {
-					return []reconcile.Request{{
-						NamespacedName: InitNamespacedName(),
-					}}
-				},
-			),
+			}, enqueueOCSInit,
```

Comment:
Please move the `enqueueOCSInit` to its own raw. future changes to the first arg should not result in changes to the second arg
```suggestion
			}, 
			enqueueOCSInit,
```